src/i18n/strings/tchap_translations.json

```diff
@@ -88,5 +88,9 @@
   "We have compiled a list of the questions our users ask the most frequently. Your question may be answered there.": {
     "en": "We have compiled a list of the questions our users ask the most frequently. Your question may be answered there.",
     "fr": "Nous avons constitué une liste des question les plus fréquentes posées par nos utilisateurs. Votre question a peut-être déjà une solution."
+  },
+  "You have made to many requests and have been blocked. Try again later.": {
+    "en": "You have made to many requests and have been blocked. Try again later.",
+    "fr": "Vous avez fait trop de requêtes et avez été bloqué. Réessayez plus tard."
```

Comment:
scary!  "requetes"->"tentatives" , "bloqué temporairement" ?

Reply:
Nous avons ceci dans la v2 : 
`Your last three login attempts have failed. Please try again in 30 minutes.`

Reply:
after testing, it's 3 attempts, and the delay for unblocking is variable, between 5 and 8 minutes. It probably depends  how many attempts you made recently ? 
So I used "Your last three login attempts have failed. Please try again in a few minutes."
Thanks for suggestions ! 👍